apollo-ios-pagination/Sources/ApolloPagination/AsyncGraphQLQueryPagerCoordinator.swift

```diff
@@ -317,7 +313,7 @@ actor AsyncGraphQLQueryPagerCoordinator<InitialQuery: GraphQLQuery, PaginatedQue
       var didFail = false
       switch fetchType {
       case .initial:
-        initialPageResult = data as? GraphQLResult<InitialQuery.Data>
+        initialPageResult = data as? GraphQLResponse<InitialQuery>
```

Comment:
Interesting: Has the definition of `GraphQLResponse` changed for 2.x? Happy to catch up and read more.